google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java

```diff
@@ -51,8 +51,21 @@
 
 @RunWith(JUnit4.class)
 public class DatabaseClientImplTest {
-  private static final String DATABASE_DOES_NOT_EXIST_MSG =
-      "Database not found: projects/<project>/instances/<instance>/databases/<database> resource_type: \"type.googleapis.com/google.spanner.admin.database.v1.Database\" resource_name: \"projects/<project>/instances/<instance>/databases/<database>\" description: \"Database does not exist.\"";
+  private static final String DATABASE_NOT_FOUND_FORMAT =
```

Comment:
nits: I feel that a hardcoded message is more clear here, because if the message in the code has been mistakenly changed, this test will let it pass instead of finding out the difference.

Reply:
Good point. I changed it back to a hard coded string.